### DISCUSSION TITLE
FDF Neon Plus sensors possible to connect?

### DISCUSSION THREAD

**Discussion by Anohito82** [JaapvanEkris/openrowingmonitor#44] [Q&A]
Hello,

I'm very happy to have found your project! Thanks a lot for the hard work you put into this.
If someone could help me out or point into the right direction, I'd be very happy.

I'd like to find out whether I could use ORM with my new rower, without replacing any sensors, in order to make it work with ANT+ for Garmin. Unfortunately I'm quite puzzled about sensors and I'm not good with hardware modifications.

I tried to find out about the installed sensor(s), but the only thing I found was the parts list and "exploded diagram" from the manufacturer, with a part number (11072) I couldn't find anything about.
But if this number has any meaning then it might be the same as in the FDF Apollo or Neon Pro rowers.

I looked at the threads linked to in the device compatibility chart regarding the 2 FDF devices listed, if I understood this correctly, this means I'd have to replace sensors, is this correct?

When I look at the cable going to the console, it's 2 pins, so I could theoretically connect it to a pi's gpio (I don't have one yet so couldn't try).
I'm wondering if the hardware might be compatible and enough to make it work, except for modifying the rower config in ORM, or if it's unlikely to succeed without replacing the sensor.

Thanks a lot for reading and sorry for these basic questions, but after reading for many hours I'm still at a loss.

**Comment by Abasz** [comment]
If its two pin, there is a high chance that its a reed switch at the other end. It would be triggered by a magnet placed, hopefully, on the shaft of the impeller. If this is the case its probably the simplest thing. You just connect one pin to the gipo pin, one to ground, apply the settings and start rowing :) (setting the correct settings will not be this easy, but hardware setup will be much easier than finding the correct settings)

**Reply by Anohito82** [reply]
Thanks so much @Abasz !! This is very encouraging :)
I took pictures in case I misunderstood about this: https://imgur.com/a/f8IGWoP

Hopefully the other water rower from FDF mentioned with "Sensor Replacement" https://github.com/JaapvanEkris/openrowingmonitor/blob/main/docs/Supported_Rowers.md had different sensors.

I guess I'll have to buy a pie to find out ;)

**Reply by Abasz** [reply]
Technically you can try a multimeter if it has continuation feature or a battery and an led and check if it blinks when magnet passes. You can imagine a reed as a simple light switch. Magnet close=conducts far=does not conduct

**Reply by Anohito82** [reply]
Thanks a lot Abasz for your expertise and answering so quickly. I might give this a try first when I can finally get to this. 
Also thanks a lot for contributing so much to this great project!

**Reply by Frits1980** [reply]
> Thanks a lot Abasz for your expertise and answering so quickly. I might give this a try first when I can finally get to this. Also thanks a lot for contributing so much to this great project!

Hi @Anohito82 , did you managed to connect your rower? I have an old first degree fitness rower which uses the same sensor. I’ve hooked up the sensor even without taking of the plug. I could press the plug straight into GPIO 26 and the ground next to it. I did edit the boot file on the rpi to make it work. Like mentioned in the installation manual of ORM.
